src/library_pthread.js

```diff
@@ -660,6 +660,18 @@ var LibraryPThread = {
     return navigator['hardwareConcurrency'];
   },
 
+  emscripten_thread_pool_size: function() {
+#if PTHREAD_POOL_SIZE
+    const pthreadPoolSize = {{{ PTHREAD_POOL_SIZE }}};
+    if (pthreadPoolSize > _emscripten_num_logical_cores()) {
+      err('PTHREAD_POOL_SIZE is bigger than emscripten_num_logical_cores()!');
+    }
```

Comment:
the warning seems too unrelated to me. that is, if this just gets the pool size, it shouldn't also validate it. if we think that warning is important, it should be present where the pool is created (and that could be in a separate PR).